src/api/include/migraphx/migraphx.hpp

```diff
@@ -493,12 +495,110 @@ struct shapes : MIGRAPHX_HANDLE_BASE(shapes), array_base<shapes>
     };
 };
 
+struct operation : MIGRAPHX_HANDLE_BASE(operation)
+{
+    operation(migraphx_operation* p, own) { this->set_handle(p, own{}); }
+
+    operation(migraphx_operation* p, borrow) { this->set_handle(p, borrow{}); }
+
+    template <class... Ts>
+    operation(const char* name, const char* attributes = nullptr, Ts... xs)
+    {
+        this->make_handle(&migraphx_operation_create, name, attributes, xs...);
+    }
+
+    std::string name()
+    {
+        std::array<char, 1024> out_name;
+        call(&migraphx_operation_name, out_name.data(), 1024, this->get_handle_ptr());
+        return {out_name.data()};
+    }
+};
+
+struct instruction_ref : MIGRAPHX_HANDLE_BASE(instruction_ref)
+{
+    instruction_ref(migraphx_instruction_ref* p, own) { this->set_handle(p, own()); }
```

Comment:
Remove the `own` parameter from the constructor and use `{}` to construct `own`: `instruction_ref(migraphx_instruction_ref* p) { this->set_handle(p, own{}); }`

Reply:
Done.